lib/config.ts

```diff
@@ -562,11 +562,9 @@ export interface Config {
     /**
      * How to format features (default: progress)
      */
-    format?: string;
-    // Other options include `coffee`, `noSnippets`, and `dryRun`
-    coffee?: any;
-    noSnippets?: any;
-    dryRun?: any;
+    format?: string[] | string;
+    // Other options include `dryRun`
```

Comment:
instead of having 
```
// Other options include `dryRun`
```

could we have a definition? 
```
/**
 * use dry run for ...
 */
dryRun?: boolean;
```